cmd/kubeadm/app/cmd/reset.go

```diff
@@ -182,6 +182,15 @@ func (r *Reset) Run(out io.Writer, client clientset.Interface) error {
 	}
 	resetConfigDir(kubeadmconstants.KubernetesDir, r.certsDir)
 
+	// Output help text instructing user how to remove iptables rules
+	msg := `
+	[reset] The reset process does not reset or clean up iptrables rules.
+	[reset] If you wish to reset iptables, you must do so manually.
+	[reset] Ex: 
+	[reset] iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
```

Comment:
What if it's using IPVS?  Does the proxy run offer a one-shot cleanup?  If so, it should hide that abstraction

Reply:
@timothysc This is what kube-proxy does: https://github.com/kubernetes/kubernetes/blob/91d6d7530382bf120a1575bae72ca36a33842e95/cmd/kube-proxy/app/server.go#L466-L473

Reply:
After reading the convo [here](https://github.com/kubernetes/kubernetes/issues/55857) and all the caveats therein, I still favor help text on the kubeadm side over wrangling/vendoring code to control cleanup over this side.

Reply:
Why don't you have the instructions call out `docker run kube-proxy ... arg`

Reply:
@timothysc OK, so `docker run -u 0 k8s.gcr.io/kube-proxy:v1.12.2 kube-proxy --cleanup` does not work. Much less without `-u 0`. /me wonders if this functionality has been e2e or manually tested. 🤔

Reply:
Would adding a line such as 

`[reset] If your system uses IPVS, please contact your system administrator for proper reset`

help?

Reply:
Or something like `[reset] If your system uses IPVS, you might want to clear them via ipvsadm --clear or similar` 

wdyt @timothysc

Reply:
PR updated with ipvs reset wording